modules/evm-accounts/src/lib.rs

```diff
@@ -149,7 +150,7 @@ pub mod module {
 		}
 
 		// TODO: weights & benchmarking
```

Comment:
I clicked merge button too fast... can you delete this TODO?

Reply:
Will do